doc/manpages/dinit.8.m4

```diff
@@ -108,6 +108,10 @@ If service description settings contain relative cgroup paths, they will be reso
 this path.
 This option is only available if \fBdinit\fR is built with cgroups support.
 .TP
+\fB\-\-disable\-selinux\fR
```

Comment:
The option was changed to `--disable-selinux-policy`, you need to update the documentation to match.